src/include/catalog/table_catalog.h

```diff
@@ -73,6 +79,15 @@ class TableCatalogObject {
   std::shared_ptr<ColumnCatalogObject> GetColumnObject(
       const std::string &column_name, bool cached_only = false);
 
+  // Evict all layouts from the cache
```

Comment:
I may remember this wrong, but last time we talked you were saying add an extra column in pg_table to record the largest layout_id for each data table.